integrations/event-handler/main.go

```diff
@@ -55,9 +58,23 @@ func main() {
 		kong.Description(pluginDescription),
 	)
 
+	logCfg := logger.Config{
+		Severity: "info",
+		Output:   "stderr",
+		Format:   "text",
+	}
 	if cli.Debug {
 		enableLogDebug()
+		logCfg.Severity = "debug"
+	}
+	log, err := logCfg.NewSLogLogger()
+	if err != nil {
+		fmt.Println(trace.DebugReport(trace.Wrap(err, "initializing logger")))
+		os.Exit(-1)
 	}
+	// Whilst this package mostly dependency injects slog, upstream dependencies
+	// may still use the default slog logger.
+	slog.SetDefault(log)
```

Comment:
Hmmm, I forgot to do this on the msteams plugin 🙃

Reply:
Yeah - I wasn't too sure if it makes more sense to have an `InitSlogger` in the loggers package similar to the `Init` method we have today for logrus.